src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -376,6 +376,18 @@ public function initialized($id)
     }
 
     /**
+     * Clears shared services from the container.
+     */
```

Comment:
We should definitely document the side effects of calling this method,  not only the intended effects (memory) but also the side effects (a shared service will be re-created if get() is called).